internal/cli/runner_install.go

```diff
@@ -308,9 +308,7 @@ func (c *RunnerInstallCommand) Run(args []string) int {
 		} else {
 			odrConfig.TargetRunner = &pb.Ref_Runner{
 				Target: &pb.Ref_Runner_Id{
-					Id: &pb.Ref_RunnerId{
-						Id: strings.ToUpper(id),
-					},
+					Id: &pb.Ref_RunnerId{Id: id},
```

Comment:
This is the one that we need to solve the issue https://github.com/hashicorp/waypoint/issues/3989, but we changed them all for consistency / so the CLI looks nice.